src/main/java/conversion/utils/HttpHelper.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Base Microservice Example
+ *
+ * Project Info: https://github.com/idrsolutions/base-microservice-example
+ *
+ * Copyright 2018 IDRsolutions
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package conversion.utils;
+
+import conversion.Individual;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Static collection of methods to help with sending http requests.
+ */
+public class HttpHelper {
+
+    private static Logger LOG = Logger.getLogger(HttpHelper.class.getName());
+    private static int MaxRetries = 1;
+
+    /**
+     * Tries to send individual json data to the callbackUrl provided by the
+     * user when the file is submitted.
+     *
+     * @param callbackUrl
```

Comment:
Everyone else has pointed out everything I was gonna say, so just a few minor things from me! :')

Params here need detailing

Reply:
Have adjusted with the latest commit